src/django/api/templates/mail/slc_contribution_rejected_body.html

```diff
@@ -21,9 +22,7 @@
         <p>
           <b>Review the feedback below:</b>
         </p>
-        <ul>
-          <li>{{ action_reason_text_raw }}</li>
-        </ul>
+        {{ action_reason_text_raw|bleach }}
```

Comment:
Should this be indented like this if possible? Just makes it a bit more readable.

```
{{ action_reason_text_raw | bleach }}
```

Reply:
Done.